### PR TITLE
DDP-6361 PanCan: i18n strings for toolkit and sdk components

### PR DESCRIPTION
**Pull request by aglushko-broad**
Add i18n strings for the rest of toolkit and sdk components 
Add Spanish translation from mbc and testboston where possible and add [ES] prefix for strings w/o translation

### CODE REVIEW

File: ddp-workspace/projects/ddp-testboston/src/assets/i18n/es.json [hunk 1]
```diff
@@ -1,635 +1,635 @@
...
-        "Message": "Utilice la herramienta de búsqueda para administrar a los participantes.",
-        "Button": "Buscar"
-      }
-    },
-    "Password": {
-      "Title": "Ingrese la contraseña.",
-      "Text": "Ingrese su contraseña.*",
-      "InputPlaceholder": "Contraseña",
-      "PasswordWrongError": "Contraseña incorrecta. Vuelva a intentarlo.",
-      "OtherError": "[ES] We're sorry, but an error has occurred. Please try again.",
```
Comment (aglushko-broad) [inline ddp-workspace/projects/ddp-testboston/src/assets/i18n/es.json:387]: I added translation for this line from mbc

File: ddp-workspace/projects/ddp-pancan/src/assets/i18n/en.json [hunk 2]
```diff
@@ -332,12 +340,243 @@
...
+            "DateNavyValidationRule": "Entered date is invalid"
+        },
+        "SubjectPanel": {
+            "Link": "Return to subject look-up",
+            "InvitationCode": "Subject Invitation Code:",
+            "Email": "Email:",
+            "Name": "Name:",
+            "ShortId": "Subject Short ID/HRUID:"
+        },
+        "AdminActionPanel": {
```
Comment (aglushko-broad) [inline ddp-workspace/projects/ddp-pancan/src/assets/i18n/en.json:577]: added translation for AdminActionPanel and es prefix for ActivityEdit

File: ddp-workspace/projects/ddp-testboston/src/assets/i18n/es.json [hunk 3]
```diff
@@ -1,635 +1,635 @@
...
+                "Placeholder": "Ingrese el código postal.",
+                "Error": "Código postal no válido.",
+                "Success": "Código postal válido."
+            },
+            "Step3": {
+                "Title": "Confirme verbalmente que el participante voluntario es mayor de 18 años antes de iniciar la inscripción."
+            }
+        }
     },
-    "AdminActionPanel": {
```
Comment (aglushko-broad) [inline ddp-workspace/projects/ddp-testboston/src/assets/i18n/es.json:627]: added translation for AdminActionPanel and es prefix for ActivityEdit

File: ddp-workspace/projects/ddp-testboston/src/assets/i18n/es.json [hunk 4]
```diff
@@ -1,635 +1,635 @@
...
-        "E.TIME_ZONE.UNAVAILABLE": "El servicio de zona horaria no está disponible en este momento.",
-        "E.PO_BOX.INTERNATIONAL": "No se puede verificar la casilla postal internacional. Tenga en cuenta que eso puede causar errores en solicitudes correctas."
-      }
-    },
-    "TooltipAlt": "Información",
-    "SubjectPanel": {
-      "Link": "Regresar a la búsqueda de participantes voluntarios",
-      "InvitationCode": "Código de invitación del participante voluntario:",
-      "Email": "Electrónico:",
-      "Name": "[ES] Name:",
```
Comment (aglushko-broad) [inline ddp-workspace/projects/ddp-testboston/src/assets/i18n/es.json:624]: added translation for Name

File: ddp-workspace/projects/ddp-pancan/src/assets/i18n/es.json [hunk 5]
```diff
@@ -23,340 +23,579 @@
...
             },
             "ParticipateSection": {
-                "Title": "Here’s how to participate",
+                "Title": "[ES]Here’s how to participate",
                 "Step1": {
-                    "Title": "STEP 1",
-                    "Time": "10-15 minutes",
-                    "Description": "Provide consent and tell us where you’ve been treated"
+                    "Title": "PASO 1",
+                    "Time": "[ES]10-15 minutes",
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-pancan/src/assets/i18n/es.json:45]: sugggest simple translation to Spanish
`10-15 minutos` 

here and for every step below

File: ddp-workspace/projects/ddp-pancan/src/assets/i18n/es.json [hunk 6]
```diff
@@ -23,340 +23,579 @@
...
-                    "Description": "Provide samples (we’ll send a kit)"
+                    "Title": "PASO 3",
+                    "Time": "[ES]10 minutes",
+                    "Description": "[ES]Provide samples (we’ll send a kit)"
                 },
                 "Step4": {
-                    "Title": "STEP 4",
-                    "Description": "Learn with us along the way"
+                    "Title": "PASO 4",
+                    "Description": "[ES]Learn with us along the way"
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-pancan/src/assets/i18n/es.json:60]: suggest 
`"Aprenda junto con nosotros"` 
( got it from ddp-mbc/es.json, see ThirdStep)

**Review by alextanlan — APPROVED** [review]

**Pull request merged**